modules/lib/src/main/resources/assets/js/app/wizard/page/contextwindow/inspect/page/PageTemplateAndControllerForm.ts

```diff
@@ -11,7 +11,7 @@ export class PageTemplateAndControllerForm
         super('page-template-and-controller-form');
 
         let fieldSet = new Fieldset();
-        fieldSet.add(new FormItemBuilder(selector).setLabel(i18n('field.page.template')).build());
+        fieldSet.add(new FormItemBuilder(selector).setLabel(i18n('field.page.renderer')).build());
```

Comment:
'template' is being shown over a dropdown even when controller is selected, thought about more common name

Reply:
That's how it's been for a very long time and no one complained. It might be "Template or controller" (but that's too long and technical). Let's roll it back to "Template".

Reply:
Ok!